client/src/components/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const Header: React.FC = () => {
+  return (
+    <header className="px-2 py-4 text-start">
+      <h1 className="text-4xl font-mono font-semibold">Cat-GPT</h1>
```

Comment:
Can you add a button here too to switch from Dark to Light

Reply:
done